mmv1/templates/terraform/examples/alloydb_cluster_basic.tf.erb

```diff
@@ -0,0 +1,19 @@
+resource "google_alloydb_cluster" "<%= ctx[:primary_resource_id] %>" {
+  provider   = google-beta
+  cluster_id = "<%= ctx[:vars]['alloydb_cluster_name'] %>"
+  location   = "us-central1"
+  network    = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}"
+
+  initial_user {
```

Comment:
It looks like this field is optional - would it be possible to leave it out of the basic test?

Reply:
ill double check but i recall it isnt

Reply:
👍  if it isn't optional then could you mark it as required in api.yaml?

Reply:
optional, removing.

Reply:
docs mention its required btw...